src/sentry/static/sentry/app/components/modals/featureTourModal.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import omit from 'lodash/omit';
+import styled from '@emotion/styled';
+
+import {openModal, ModalRenderProps} from 'app/actionCreators/modal';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import {t} from 'app/locale';
+import {IconClose} from 'app/icons';
+import {callIfFunction} from 'app/utils/callIfFunction';
+import space from 'app/styles/space';
+
+type TourStep = {
+  title: string;
+  image?: React.ReactNode;
```

Comment:
I started wondering if we should prefer React.ReactElement over React.ReactNode.. because a ReactNode can be also a string ...